content/_index.md

```diff
@@ -1,86 +1,137 @@
-- [Dan Blah](https://github.com/danblah) @ [Reset](https://www.reset.tech/)
-- [Fredrik Jonsson](https://github.com/frjo) @ [Combonetwork](https://www.combonet.se/)
+## Create single or multiple rounds
+
+You can create single or multiple application rounds for your fund, depending on your organisations needs.
+
+## Build forms
+
+With Hypha you can create all the forms needed for your fund: application, review, deciding forms etc.
+
+## Review and analyse submissions
+
+Applications can be reviewed by internal, external advisors, or both.
+
+## Communicate with applicants
+
+Once the review process has completed Hypha helps you manage communications between you and applicants.
+
+## Manage post-application process
+
+You can also manage the post-application process important tasks using Hypha. We will be expanding to support invoicing and payments.
+
+## Automate workflows and integrate with other services
+
+You can automate repeated workflows. You can also integrate Hypha with other automation and notification services (Slack for notifications, Mailgun for email communications, etc)
+
+# Data ownership - your data is yours
+
+Unlike other submission management platforms Hypha is self-hosted.
+
+This means that the applications, and all associated information created is available to you and your applicants on your own systems - not handed over to a company where it can be difficult to analyse the data.
+
+# Privacy is important
+
+The Hypha code is regularly audited. It has many privacy and security features.
+
+# Benefit from the powers of open source
+
+Hypha is open source software. This means the license invites you to expand what can do Hypha, create add-on functionality not already available.
 
+All we ask is you contribute these improvements back to the project so others can benefit.
 
-## Community
+# Technology
 
-### Contributors
+Hypha itself is built on many open source software projects. We use [Django](https://www.djangoproject.com/), [Wagtail](https://wagtail.io/), and [React](https://reactjs.org/).
 
-Thank you to all the people who have contributed code to Hypha <3
+View our [Roadmap](https://github.com/OpenTechFund/hypha/wiki/Roadmap) for upcoming features and enhancements.
 
+
+## Building Hypha
+
+Hypha [source code is hosted on Github](https://github.com/HyphaApp/) and [development takes](https://github.com/HyphaApp/hypha/issues) place in the open.
+
+Hypha uses the [BSD 3-clause](./LICENSE) license.
+
+[![<OpenTechFund>](https://circleci.com/gh/OpenTechFund/hypha.svg?style=shield)](https://circleci.com/gh/OpenTechFund/hypha)
+
+## Deploying Hypha
+
+To support organisations having ownership over their data, Hypha can be deployed [on Heroku](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Heroku), [using Docker](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Docker), or [ with your own server](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Standalone-App).
```

Comment:
The extra space before "with" in `[ with your own` should be removed.

Reply:
👍